cpp/include/cudf/binaryop.hpp

```diff
@@ -33,7 +33,7 @@ namespace cudf {
 /**
  * @brief Types of binary operations that can be performed on data.
  */
-enum class binary_operator : int32_t {
+enum class binary_op : int32_t {
```

Comment:
I think this should be `binary_operator`, not `binary_op`. These are specifically operators, and "op" may be ambiguous. Instead, change `unary_op` to `unary_operator` (which I have a hunch will be a smaller PR)

Reply:
Done.